features/fixtures/rails_integrations/app/config/environments/development.rb

```diff
@@ -44,4 +44,6 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.hosts = []
```

Comment:
The default Rails hosts stop MR from accessing the fixture using the docker network (i.e. at `http://rails_integrations:3000`) — https://guides.rubyonrails.org/configuring.html#configuring-middleware

Reply:
Perhaps the line deserves that as a comment above it in the source file for posterity?